### DISCUSSION TITLE
Camera access failed

### DISCUSSION THREAD

**Discussion by TimNe0** [alienatedsec/yi-hack-v5#250] [General]
Had a successfully paired YI Dome 1080p then about a week ago it stopped responding in the app.  The wife helpfully deleted the pairing from the app.

I can reset with no problems and it connects to the wifi.  I can access it from the web interface.  I cannot pair it with the latest version of the android app.  Is this a known fault and is there any way of resolving this to work with the offical app? If not, is there a 3rd party app which I can use and control it with?  In the meantime I've bought a different yi camera and paired it with the app, so it's not super urgent, just annoying.

**Comment by TimNe0** [comment]
HTTPD=yes
TELNETD=yes
SSHD=yes
FTPD=yes
BUSYBOX_FTPD=no
DISABLE_CLOUD=no
REC_WITHOUT_CLOUD=no
MQTT=no
RTSP=yes
RTSP_STREAM=high
RTSP_AUDIO=yes
RTSP_AUDIO_NR_LEVEL=0
ONVIF=yes
ONVIF_WSDD=yes
ONVIF_PROFILE=high
ONVIF_WM_SNAPSHOT=yes
NTPD=yes
NTP_SERVER=pool.ntp.org
PROXYCHAINSNG=no
RTSP_PORT=554
ONVIF_PORT=80
HTTPD_PORT=8080
USERNAME=
PASSWORD=
FREE_SPACE=0
FTP_UPLOAD=no
FTP_HOST=
FTP_DIR=
FTP_DIR_TREE=no
FTP_USERNAME=
FTP_PASSWORD=
FTP_FILE_DELETE_AFTER_UPLOAD=yes
SSH_PASSWORD=
CRONTAB=

**Comment by alienatedsec** [comment]
@TimNe0 Sorry for the late response - try resetting (holding a pin for 5 secs) the camera.
